### ISSUE TITLE
Modify item show pages so media opens in lightbox like viewer?

### ISSUE THREAD

**Issue by mebrett** [chnm/hearing-the-americas#20]
At present image media opens in a new window. Consider changing so it cannot be opened, or try and implement a pop-open viewer ([see this forum thread](https://forum.omeka.org/t/lightbox-for-items-show-page-for-omeka-s/13920)) if does not break accessibility.
